src/antsibull/constants.py

```diff
@@ -22,3 +22,7 @@
 DOCSITE_COMMUNITY_URL = "https://docs.ansible.com/ansible/latest/community"
 
 COLLECTION_EXCLUDE_DIRS = ("docs", "tests")
+
+CORE_PORTING_GUIDE_BASE_URL = (
+    "https://raw.githubusercontent.com/ansible/ansible-documentation/devel"
```

Comment:
```suggestion
    "https://raw.githubusercontent.com/ansible/ansible-documentation/{tag}"
```
and replace `{tag}` by `devel` in the code. It should be the release's tag in the future, but for that we really need https://github.com/ansible/ansible-documentation/issues/66.

Reply:
Indeed. I'd like to do this, but we should see how https://github.com/ansible/ansible-documentation/issues/66 shakes up first.